bundles/NewsletterBundle/config/pimcore/default.yaml

```diff
@@ -0,0 +1,45 @@
+pimcore_newsletter:
+    source_adapters:
+        defaultAdapter: pimcore_newsletter.document.newsletter.factory.default
+        csvList: pimcore_newsletter.document.newsletter.factory.csv
```

Comment:
yes, it was removed [here](https://github.com/pimcore/pimcore/pull/14848/files#diff-57a744cb3711a73caaa4c220da3360071ad3781f63f95987f8ccbdb7fb808d7e) and not added to the NewsletterBundle.

Reply:
optimized this a bit to load custom reports adapter only when custom reports bundle is installed https://github.com/pimcore/pimcore/pull/14848/commits/c50095bd53e5d35d03f122d088cdb00bd542f060

Reply:
look better now but there's something that is missing or so
![image](https://user-images.githubusercontent.com/6014195/231157525-f0941620-16e4-4a5b-b41f-bb293d76614e.png)
because the problem `Cannot send newsletters because Address Source Adapter with identifier reportAdapter could not be found` is back again

Reply:
yes, it seems that is not affected by the order in `bundles.php`